primitives/core/src/sr25519.rs

```diff
@@ -611,6 +611,45 @@ impl CryptoType for Pair {
 	type Pair = Pair;
 }
 
+
+/// Batch verification.
```

Comment:
This doc comment is really no more useful than the function name itself.. :(

Reply:
added